github/ipld.yml

```diff
@@ -131,6 +131,9 @@ repositories:
     allow_update_branch: false
     archived: false
     collaborators:
+      admin:
+        - rvagg
+        - vmx
```

Comment:
No particular objection, but wouldn't it be better to fix this by giving say the Go/JS/Rust teams push access to the repo?

If those teams are too wide then probably they should be reduced in general since this seems like a repo you'd want to push people maintaining IPLD libraries towards using.